app/mailers/invoice_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class InvoiceMailer < ApplicationMailer
+  default from: 'test@test.com'
+
+  def notify_user_email(user)
```

Comment:
Thinking if we can use a more assertive name like `invoice_about_to_expire` 🤔 Ideas?